CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Matterpoll
+
+Thank you for your interest in contributing! Join the [**Matterpoll channel**](https://community.mattermost.com/core/channels/matterpoll) on the Mattermost community server for discussion about this plugin.
+
+
+## Reporting Issues
+
+If you think you found a bug, [please use the GitHub issue tracker](https://github.com/matterpoll/matterpoll/issues/new?template=bug_report.md) to open an issue. When opening an issue, please provide the required information in the issue template.
+
+
+## Translating strings
+
+Matterpoll supports localization to various languages. We as maintainers rely on contributors to help with the translations.
+
+Matterpoll uses [go-i18n](https://github.com/nicksnyder/go-i18n) as library and tool to manage translation. The CLI tool `goi18n` is required to manage translation. You can install it by running `env GO111MODULE=off go get -u github.com/nicksnyder/go-i18n/v2/goi18n`.
+
+The localization process is defined below:
+- During development, new translation strings may be added or existing ones updated.
+- When a new version is planned to release soon, a repository maintainer opens an issue informing about this. The maintainer will ping all translation maintainer to inform them about this.
+- Translation maintainers submit PRs with new translations, which may get reviewed by other translators.
+- After all translation PRs are merged, the new version is released.
```

Comment:
@hanzei It was about the translations PRs and how long we'd wait for translation maintainers to submit them

Reply:
Please see my answer at https://github.com/matterpoll/matterpoll/pull/166#discussion_r295316022